src/CommandLine.hs

```diff
@@ -36,6 +38,7 @@ defaultCommandLineArgs = CommandLineArgs
     , progressColour = Auto
     , progressInfo   = Brief
     , splitObjects   = False
+    , buildRoot      = UserSettings.userBuildRoot
```

Comment:
BTW, I think you can just move the `_build` from `UserSettings` to here.

Reply:
Indeed, one less hoop to go through to figure out the default value. I'll make that change in an upcoming commit.